### PR TITLE
feat: update website to new design

### PR DESCRIPTION
**Pull request by moripen**

### CODE REVIEW

File: src/components/layout/footer/footer.tsx [hunk 1]
```diff
@@ -1,43 +1,33 @@
 import { Box, Container, Flex, Grid, Link, Text } from 'theme-ui';
-import { Booking } from '../../booking';
-import { Logo } from '../../logo';
-import { NetworksLinks } from './networks-links';
 
 export const Footer: React.FC = () => (
-  <Box sx={{ minHeight: '5vh', background: 'black80', mt: 'auto' }}>
-    <Container variant="readable">
+  <Box sx={{ minHeight: '5vh', background: '#0F2040', mt: 'auto' }}>
```
Comment (braaar) [inline src/components/layout/footer/footer.tsx:4]: Why are we setting a hex color here? We should use a value from the theme

File: src/pages/about.tsx [hunk 2]
```diff
@@ -106,13 +106,21 @@ const About: React.FC = () => {
...
             }
           />
+          <ContactPerson
+            name={'Markus Moripen'}
+            photoPath={'/persons/markus.jpg'}
+            phone={'+47 970 30 084'}
+            email={'markus@bjerk.io'}
+            title={'Programvareutvikler'}
+            github={'https://github.com/CalculatedCode'}
+            />
```
Comment (braaar) [inline src/pages/about.tsx:123]: Separate this out to a different PR?

File: src/components/layout/footer/footer.tsx (hunk 1 above)
Comment (CalculatedCode) [reply src/components/layout/footer/footer.tsx:4]: Fixed!

**Comment by braaar** [comment]
Perhaps we should call `Logo1.svg`, etc. "Element" instead of "Logo"?

**Comment by braaar** [comment]
Also, I think all lowercase file names are easier to deal with, in general

**Comment by CalculatedCode** [comment]
Agreed!

**Comment by CalculatedCode** [comment]
New version comitted with the new names - now "element".

File: figmagic.json [hunk 3]
```diff
@@ -0,0 +1,8 @@
+{
```
Comment (cobraz) [inline figmagic.json:1]: This change seems unrelated. IMHO, we should remove figmatic from `@bjerkio/brand`.

File: src/components/layout/footer/footer.tsx [hunk 4]
```diff
@@ -1,43 +1,30 @@
...
-        sx={{ flexDirection: 'column', py: 5, gap: 5, color: 'background' }}
+        sx={{
+          color: 'background',
+          justifyContent: 'space-between',
+          alignItems: 'center',
+          fontStyle: 'normal',
+          fontWeight: 400,
+          lineHeight: '23px',
+          letterSpacing: -0.02,
+          backgroundSize: 'cover',
```
Comment (cobraz) [inline src/components/layout/footer/footer.tsx:15]: This seems like a lot to handle in a single Flex component. Maybe consider moving it to theme?

File: src/components/layout/header.tsx [hunk 5]
```diff
@@ -1,37 +1,57 @@
 import { Link, NavLink } from 'next-theme-ui';
 import { useRouter } from 'next/router';
-import { Container, Flex } from 'theme-ui';
+import { Box, Container, Flex } from 'theme-ui';
+import { Booking } from '../booking';
 import { Logo } from '../logo';
 
 export const Header: React.FC = () => {
   const { pathname } = useRouter();
+  const color = pathname === '/' ? 'green100' : 'green20';
```
Comment (cobraz) [inline src/components/layout/header.tsx:9]:
```suggestion
  const isHome = pathname === '/';
  const color = isHome ? 'green100' : 'green20';
```

File: src/components/layout/header.tsx [hunk 6]
```diff
@@ -1,37 +1,57 @@
...
+            />
+          </Link>
+          <NavLink
+            href={'/about'}
             sx={{
-              maxWidth: '100px',
-              cursor: 'pointer',
-              height: '40px',
-              color: 'text',
+              color: pathname.includes('about') ? '#2A3BCC' : 'text',
```
Comment (cobraz) [inline src/components/layout/header.tsx:33]: It's probably better to add `#2A3BCC` to the theme?

File: src/pages/about.tsx [hunk 7]
```diff
@@ -1,127 +1,118 @@
 import { Link } from 'next-theme-ui';
```
Comment (cobraz) [inline src/pages/about.tsx:1]: Seems like this file had some unicode issues.

https://github.blog/changelog/2021-10-31-warning-about-bidirectional-unicode-text/

File: src/components/container-with-background.tsx [hunk 8]
```diff
@@ -0,0 +1,24 @@
+import { Box, Container } from 'theme-ui';
+
+export interface ContainerWithBackgroundProps {
+  color?: 'green' | 'beige';
+}
+
+export const ContainerWithBackground: React.FC<
```
Comment (cobraz) [inline src/components/container-with-background.tsx:7]: Do we need this? Does it need to be two components?

**Review by cobraz — COMMENTED** [review]
We should probably move illustrations/svgs into JSX. AFAIK, our logo already exist as a JSX.

Maybe @AnneMatilde can name our illustrations too, so we can get more sensible names than element 1,2,3? Maybe some of the variations could be programmable?

**Comment by braaar** [comment]
A bit too much space here on small screens. The header looks a bit off as well. 

Padding values on  `ContainerWithBackground` could use some breakpoints.

Screenshot 2022-06-27 at 13 23 23

**Comment by AnneMatilde** [comment]
Hi! Great work so far. I have a few suggestions for desktop:
 
image
- The design elements should be fixed to the edge of the view

- Footer should be a bit taller, maybe double the height

image
- Add a bit more padding to left and right for the button

image
- Change the hover color to purple instead of green, and ad an underline (underline on hover for all links)

image
- Name is a litte too bold

image
- Add linkes to the e-mail and phone-number

**Comment by AnneMatilde** [comment]
> A bit too much space here on small screens. The header looks a bit off as well.
> 
> Padding values on `ContainerWithBackground` could use some breakpoints.
> 
> Screenshot 2022-06-27 at 13 23 23

For the header on mobile I would suggest doing a hamburger-icon. I can ad design in Figma.

**Comment by cobraz** [comment]
> Design for mobile / header image
> 
> I this design I have adjusted the font sizes too.

https://theme-ui.com/guides/responsive-typography

File: src/components/layout/footer/footer.tsx (hunk 4 above)
Comment (CalculatedCode) [reply src/components/layout/footer/footer.tsx:15]: Agreed, but it was mostly superfluous code so shortened to:
          color: 'background',
          justifyContent: 'space-between',
          alignItems: 'center',

File: src/components/layout/header.tsx (hunk 5 above)
Comment (CalculatedCode) [reply src/components/layout/header.tsx:9]: Ah, nice! This is better indeed.

File: src/components/layout/header.tsx (hunk 6 above)
Comment (CalculatedCode) [reply src/components/layout/header.tsx:33]: It was added to the theme as 'blue100', but missed this in the sweep of hexcodes.

File: src/components/container-with-background.tsx (hunk 8 above)
Comment (CalculatedCode) [reply src/components/container-with-background.tsx:7]: We use this container on different backgrounds so it seems reasonable to have this color check IMO. Might be some other way to do it though!

File: src/pages/about.tsx (hunk 7 above)
Comment (CalculatedCode) [reply src/pages/about.tsx:1]: Hmm, there was a weird unicode character in one of the ContactPerson entries that I removed - but I am unsure how to fix this.

**Comment by CalculatedCode** [comment]
> Hi! Great work so far. I have a few suggestions for desktop:
> 
> image
> 
> * The design elements should be fixed to the edge of the view
> * Footer should be a bit taller, maybe double the height
> 
> image
> 
> * Add a bit more padding to left and right for the button
> 
> image
> 
> * Change the hover color to purple instead of green, and ad an underline (underline on hover for all links)
> 
> image
> 
> * Name is a litte too bold
> 
> image
> 
> * Add linkes to the e-mail and phone-number

Everything but the images fixed to the side of the page should now be correct. The links was already working as far as I could tell?

File: src/components/about/contact-person.tsx [hunk 9]
```diff
@@ -22,61 +20,65 @@ export const ContactPerson: React.FC<ContactPersonProps> = ({
...
         >
           <Image
             src={photoPath}
             sx={{
               minWidth: '100%',
               minHeight: '100%',
-              width: 'auto',
-              height: 'auto',
+              width: '459px',
+              height: '527px',
```
Comment (cobraz) [inline src/components/about/contact-person.tsx:35]:
```suggestion
              maxWidth: '459px',
```

Maybe this can be changed to an aspect ratio?

File: src/components/page-section.tsx [hunk 10]
```diff
@@ -0,0 +1,25 @@
...
+  children,
+  backgroundColor: backgroundColorProp,
+}) => {
+  const backgroundColor =
+    backgroundColorProp === 'green' ? 'green100' : 'green20';
+  return (
+    <Box
+      sx={{
+        backgroundColor,
+        py: ['98px', '194px'],
```
Comment (cobraz) [inline src/components/page-section.tsx:17]:
```suggestion
        py: ['6.125rem', '12.125rem'],
```

Maybe we can change these to `rem` values?

File: src/pages/about.tsx (hunk 7 above)
Comment (cobraz) [reply src/pages/about.tsx:1]: Let's just ignore it :)

File: src/pages/about.tsx [hunk 11]
```diff
@@ -1,128 +1,114 @@
...
     <Layout>
-      <Grid sx={{ gap: 5 }}>
-        <Grid sx={{ gap: 3 }}>
-          <Heading variant="title">Om oss</Heading>
-          <Paragraph variant="excerpt">
+      <PageSection>
+        <BackgroundGraphic
+          element="element2"
+          positionx="left"
+          positiony="1040px"
```
Comment (cobraz) [inline src/pages/about.tsx:15]:
```suggestion
          positiony="1040px"
```

This seems unsafe

File: src/pages/about.tsx [hunk 12]
```diff
@@ -1,128 +1,114 @@
...
+        />
+
+        <Container variant="readable">
+          <Heading variant="titlenobold" sx={{ pb: '40px' }}>
```
Comment (cobraz) [inline src/pages/about.tsx:19]:
```suggestion
          <Heading variant="titlenobold" sx={{ pb: '40px' }}>
```

This and the one below 👇 should probably be replaced with the size array.

File: src/components/about/contact-person.tsx (hunk 9 above)
Comment (moripen) [reply src/components/about/contact-person.tsx:35]: Funka med maxHeight!

File: src/components/layout/header/cross.tsx [hunk 13]
```diff
@@ -0,0 +1,29 @@
+import { FC, SVGProps } from 'react';
+import { ThemeUIStyleObject } from 'theme-ui';
+
+type Props = ThemeUIStyleObject & SVGProps<SVGSVGElement>;
+
+export const Cross: FC<Props> = () => {
```
[the remark this answers is not in the document]
Comment (moripen) [reply src/components/layout/header/cross.tsx:6]: Done!

File: src/components/page-section.tsx (hunk 10 above)
Comment (moripen) [reply src/components/page-section.tsx:17]: Not sure why, but this change actually impacts the layout of the page... maybe we can take a look together?
